src/blocks/modal.css

```diff
@@ -150,8 +151,30 @@
   transition: background 0.3s;
 }
 
-.modal__submit-btn_disabled {
-  opacity: 0.5;
+/* modifiers for delete and cancel how to style #delete modal flex container */
```

Comment:
* The `save` button doesn't change its state if the form is invalid https://snipboard.io/RyUaSu.jpg  Please, change it also in `modal.css`


`modal__submit-btn_disabled` should be returned back in `modal.css`



```css


.modal__submit-btn_disabled {
  opacity: 0.5;
}

```